scripts/npm/link/index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var path = require('path'),
+    fs = require('fs'),
+    componentsDir = path.join(__dirname, '..', '..', '..', 'src'),
+    components = getDirectories( componentsDir ),
+    exec = require( 'child-process-promise' ).exec;
```

Comment:
@anselmbradford entirely agree, but the other npm scripts are written this way. Figured it was better to be consistent. Thoughts.